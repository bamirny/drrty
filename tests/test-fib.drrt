def int fib(int x){
	if (x<2){
		return 1;
	}
	return fib(x-1) + fib(x-2);
}

def int main() {
	print(fib(1));
	print(fib(5));
	
	return 0;
}