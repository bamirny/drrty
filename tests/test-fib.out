1<br>
8<br>
